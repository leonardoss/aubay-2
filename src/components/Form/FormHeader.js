import React from 'react';

class FormHeader extends React.Component {
    render() {
        return(
            <header>
                <h3 className="title">Log in to our site</h3>
                <p className="text">Enter username or password to log on</p>
            </header>
        );
    }
}

export default FormHeader;