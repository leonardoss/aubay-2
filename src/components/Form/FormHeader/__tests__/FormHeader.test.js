import React from 'react';
import FormHeader from '../FormHeader';
import renderer from 'react-test-renderer';

it('FormHeader should renders correctly', () => {
  const tree = renderer.create(<FormHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});