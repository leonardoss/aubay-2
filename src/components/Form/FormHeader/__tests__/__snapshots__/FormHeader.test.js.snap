// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormHeader should renders correctly 1`] = `
<header>
  <h3
    className="title"
  >
    Log in to our site
  </h3>
  <p
    className="text"
  >
    Enter username or password to log on
  </p>
</header>
`;
